match/lib/match/encrypt.rb

```diff
@@ -89,8 +89,16 @@ def crypt(path: nil, password: nil, encrypt: true)
       command << "-out #{tmpfile.shellescape}"
       command << "-a"
       command << "-d" unless encrypt
-      command << "&> /dev/null" unless FastlaneCore::Globals.verbose? # to show show an error message is something goes wrong
-      success = system(command.join(' '))
+      command << "&> /dev/null" unless FastlaneCore::Globals.verbose? # to show an error message if something goes wrong
+
+      _out, err, st = Open3.capture3(command.join(' '))
+      success = st.success?
+
+      # Ubuntu `openssl` does not fail on failure
+      # but at least outputs an error message
+      if err.to_s != ''
```

Comment:
you should use `unless err.ro_s.empty?`

Reply:
Thanks, I'm pretty noob in ruby. Changed it.